multiqc/modules/sexdeterrmine/sexdeterrmine.py

```diff
@@ -128,6 +130,33 @@ def read_count_barplot(self):
             plot = bargraph.plot(self.sexdet_data, cats, config)
         )
 
+    def snp_rate_scatterplot(self):
+        """ Make a scatter plot showing relative coverage on X and Y chr
+        """
+        data = OrderedDict()
+        for sample in self.sexdet_data:
+            try:
+                data[sample] = {'x': self.sexdet_data[sample]['RateX'], 'y': self.sexdet_data[sample]['RateY']}
+            except KeyError:
+                pass
+
+        config = {
+            'id': 'sexdeterrmine-rate-plot',
+            'title': 'SexDetErrmine: Relative coverage',
+            'ylab': 'Relative Cov. on Y',
+            'xlab': 'Relative Cov. on X',
+            'xmax': 1.8,
+            'ymax': 1.2
+        }
+
+        self.add_section(
+            name = 'Relative Coverage',
+            anchor = 'sexdeterrmine-rates',
+            description = 'The coverage on the X vs Y chromosome, relative to coverage on the Autosomes.',
+            plot = scatter.plot(data, config)
+        )
```

Comment:
Please add a check that you have data before adding the section:
```suggestion
        if len(data) > 0:
            self.add_section(
                name = 'Relative Coverage',
                anchor = 'sexdeterrmine-rates',
                description = 'The coverage on the X vs Y chromosome, relative to coverage on the Autosomes.',
                plot = scatter.plot(data, config)
            )
```